### ISSUE TITLE
Reuse of OBS Virtual Cam for AR Apps

### ISSUE THREAD

**Issue by zugende** [CatxFish/obs-virtual-cam#93]
Hi,

I'm thinking about integrating OBS virtual cam into my own application installation flow to potentially create an app like "Snap Camera" (https://snapcamera.snapchat.com/). Using ffmpeg how can I send my own video stream to the virtual device? Maybe I am overseeing something but would a command like this be possible?

`ffmpeg -re -i input.mp4 -map 0:v -f mjpeg OBS-Camera`

Any help or guidance would be awesome!

**Comment by GrushBrush** [comment]
Yes, is there anyway to use the directshow filters of this framework without using OBS to choose the source?
